Code Examples/Volodymyr Ostrovskyi/Framework.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NunitHomework
+{
+    public static class Framework
+    {
+        //Copy directory from fromPath to toPath
+        public static void CopyDirectory(string fromPath, string toPath)
+        {
+            if (!System.IO.Directory.Exists(toPath))
+            {
+                System.IO.Directory.CreateDirectory(toPath);
+            }
+            if (System.IO.Directory.Exists(fromPath))
+            {
+                string[] files = System.IO.Directory.GetFiles(fromPath);
+                foreach (string s in files)
+                {
+                    string fileName = Path.GetFileName(s);
```

Comment:
@mehhrunes Can we rewrite this line to match the standards? (Fake comment, no issues ;))

Reply:
@andrii-solaris sure mate, we can do it, Indian style (fake comment by the way)